public/tier1/utlstringtoken.h

```diff
@@ -14,42 +14,116 @@
 #endif
 
 #include <limits.h>
+#include <string.h>
+#include "bufferstring.h"
```

Comment:
These includes introduce cyclic include dependency issues which causes some parts of the code to fail. As per the comments below these are better removed from here anyway!

Reply:
Solved it in the slightly different way

Reply:
I don't think what you did is sane enough, moving stuff around like that to satisfy the needs isn't ideal especially for stuff that always been there, please, just remove the utlstring and bufferstring from stringtoken as that will fix every issue related to this.